src/backend/distributed/executor/multi_utility.c

```diff
@@ -384,6 +375,11 @@ multi_ProcessUtility(Node *parsetree,
 		SetUserIdAndSecContext(savedUserId, savedSecurityContext);
 	}
 
+	if (ddlJob != NULL)
+	{
+		ExecuteDistributedDDLJob(ddlJob);
+	}
+
 	/* we run VacuumStmt after standard hook to benefit from its checks and locking */
 	if (IsA(parsetree, VacuumStmt))
```

Comment:
Passing on this, though it would be easy to do once #1287 is merged (since it adds a flag to ensure a particular `DDLJob` runs outside of a transaction).